src/hooks/VoicePlayer/index.tsx

```diff
@@ -128,7 +131,7 @@ export const VoicePlayerProvider = ({
         .then((blob) => {
           const audioFile = new File([blob], VOICE_MESSAGE_FILE_NAME, {
             lastModified: new Date().getTime(),
-            type: VOICE_MESSAGE_MIME_TYPE,
+            type: getParsedMimeType(audioFileMimeType),
```

Comment:
nit: When creating a file with a fetched blob, the file name contains an extension. Won't that be confusing later?

<img width="268" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/cc7b734c-e15b-4d83-9f88-54d9526cd207">

Reply:
Oh yeah right. So I made it differentiate the name too. https://github.com/sendbird/sendbird-uikit-react/pull/1123/commits/04f1d6b333c6e35a2739540c3f3ed5b50853422a